specification/storage/Microsoft.BlobStorage/routes.tsp

```diff
@@ -610,10 +603,6 @@ namespace Storage.Blob {
         @header("x-ms-range-get-content-crc64")
         rangeGetContentCrc64?: boolean,
 
-        /** Specifies the response content should be returned as a structured message and specifies the message schema version and properties. */
-        @header("x-ms-structured-body")
-        structuredBodyType?: string,
-
```

Comment:
`download` blob API
This removes the parameter on download API to return result as structured message
![image](https://github.com/user-attachments/assets/e3e00a6c-35b5-4eb8-84b0-a30516c652ce)

Reply:
Was the parameter incorrect?

Reply:
Hi Joel, for context-- the parameter is not incorrect but structured message is only for content validation (which is NYI) and there is discussion internally on Storage's end on how we want to implement content validation- so we decided it would be best to not expose the current impl details to customers especially since it may not represent the final cut.